tests/unit/modules/test_network.py

```diff
@@ -26,6 +26,9 @@
 from salt.exceptions import CommandExecutionError
 from salt._compat import ipaddress
 
+import logging
```

Comment:
Move logging import to the top where the other python imports are(if any).

Just because I'm nitpicking :)